app/models/album.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const logger = require('../logger'),
+  errors = require('../errors');
+
+module.exports = (sequelize, DataTypes) => {
+  const Album = sequelize.define(
```

Comment:
Rename this model to AlbumUser or something like that, other way seems like a table of albums